### ISSUE TITLE
'Register' link is 404'd

### ISSUE THREAD

**Issue by d8aninja** [RunestoneInteractive/RunestoneComponents#551]
When you click on 'register' to make an account, a 404 error presents.

**Comment by d8aninja** [comment]
Ah - you have to click on the RunestoneAcademy link in the bottom left notification about the site moving.  Would be nice for the other Register link to notify of this.

**Comment by bnmnetp** [comment]
Yes. Registrations are disabled on interactivepython.org.  Please use https://Runestone.academy

**Comment by bnmnetp** [comment]
Where did you click on a link that gave you the 404?  I'll fix it, but not sure where you are referring to.
